src/assets/styles/_variables.scss

```diff
@@ -39,3 +40,11 @@
   --KCardBackground: var(--white);
   --KCardBorder: 1px solid var(--grey-300);
 }
+:root.is-fullscreen {
+  --AppHeaderHeight: 0;
+  --AppSidebarWidth: 0;
+  --AppGap: 0;
```

Comment:
**Suggestion**: I suggest to introduce a new property `--AppContentPadding` specifically for the use on `.app-main-content`. In hindsight, I think it was a mistake to use `--AppGap` for this element in particular. The reason for that is that it’s used in a couple of other places to, for example, space out cards.

So then we have:

- `--AppContentPadding` for `.app-main-content`
- `--AppGap` for the current uses

Reply:
Oh hey, sorry for the delay, totally didn't realize this was here! This has been changed now, lmk!